projects/plugins/jetpack/modules/custom-css/custom-css-4.7.php

```diff
@@ -509,43 +533,46 @@ public static function wp_enqueue_scripts() {
 	 * @return mixed|string
 	 */
 	public static function sanitize_css( $css, $args = array() ) {
-		$args = wp_parse_args( $args, array(
-			'force'        => false,
-			'preprocessor' => null,
-		) );
+		$args = wp_parse_args(
+			$args,
+			array(
+				'force'        => false,
+				'preprocessor' => null,
+			)
+		);
 
 		if ( $args['force'] || ! current_user_can( 'unfiltered_html' ) ) {
 
 			$warnings = array();
 
 			safecss_class();
-			$csstidy = new csstidy();
+			$csstidy           = new csstidy();
 			$csstidy->optimise = new safecss( $csstidy );
 
-			$csstidy->set_cfg( 'remove_bslash',              false );
-			$csstidy->set_cfg( 'compress_colors',            false );
-			$csstidy->set_cfg( 'compress_font-weight',       false );
-			$csstidy->set_cfg( 'optimise_shorthands',        0 );
-			$csstidy->set_cfg( 'remove_last_;',              false );
-			$csstidy->set_cfg( 'case_properties',            false );
+			$csstidy->set_cfg( 'remove_bslash', false );
+			$csstidy->set_cfg( 'compress_colors', false );
+			$csstidy->set_cfg( 'compress_font-weight', false );
+			$csstidy->set_cfg( 'optimise_shorthands', 0 );
+			$csstidy->set_cfg( 'remove_last_;', false );
+			$csstidy->set_cfg( 'case_properties', false );
 			$csstidy->set_cfg( 'discard_invalid_properties', true );
-			$csstidy->set_cfg( 'css_level',                  'CSS3.0' );
-			$csstidy->set_cfg( 'preserve_css',               true );
-			$csstidy->set_cfg( 'template',                   dirname( __FILE__ ) . '/csstidy/wordpress-standard.tpl' );
+			$csstidy->set_cfg( 'css_level', 'CSS3.0' );
+			$csstidy->set_cfg( 'preserve_css', true );
+			$csstidy->set_cfg( 'template', __DIR__ . '/csstidy/wordpress-standard.tpl' );
 
 			// Test for some preg_replace stuff.
 			{
 				$prev = $css;
-				$css = preg_replace( '/\\\\([0-9a-fA-F]{4})/', '\\\\\\\\$1', $css );
+				$css  = preg_replace( '/\\\\([0-9a-fA-F]{4})/', '\\\\\\\\$1', $css );
 				// prevent content: '\3434' from turning into '\\3434'.
 				$css = str_replace( array( '\'\\\\', '"\\\\' ), array( '\'\\', '"\\' ), $css );
-				if ( $css !== $prev ) {
-					$warnings[] = 'preg_replace found stuff';
-				}
+			if ( $css !== $prev ) {
+				$warnings[] = 'preg_replace found stuff';
+			}
 			}
```

Comment:
The indenting was technically correct before. But really what should be done here is
```suggestion
			$prev = $css;
			$css  = preg_replace( '/\\\\([0-9a-fA-F]{4})/', '\\\\\\\\$1', $css );
			// prevent content: '\3434' from turning into '\\3434'.
			$css = str_replace( array( '\'\\\\', '"\\\\' ), array( '\'\\', '"\\' ), $css );
			if ( $css !== $prev ) {
				$warnings[] = 'preg_replace found stuff';
			}
```